### DISCUSSION TITLE
Multiple simultaneous time tracking

### DISCUSSION THREAD

**Discussion by ghost** [kimai/kimai#2180] [Questions & Answers]
Is it possible to configure Kimai in this way?
We have small factory, access to one computer on the shop floor. I don't want the 6-8 worker to have to log in and out whenever they change jobs which can be every 15 minutes or could be all day. So can we create a single user called "Factory" and keep that logged in. Then worker 1 comes to the computer and starts the timer for the job they are working on (project? tag?) 2 minutes later, worker 2 comes and starts timer for their job . As far as I can see at the moment, worker 2 will effectively stop the timer for worker 1 which is no good. Can both (and more) timers work at the same time?

**Comment by kevinpapst** [comment]
Yes, you can use multiple timers for one account.

Its described here: https://www.kimai.org/documentation/timesheet.html#limit-active-entries
But forget the configuration file example (it is outdated). 

You find the configuration under "Settings":
<img width="1184" alt="Bildschirmfoto 2020-12-09 um 15 14 10" src="https://user-images.githubusercontent.com/533162/101640814-42980680-3a31-11eb-91fa-32f85374ffc8.png">

**Comment by ghost** [comment]
Great thanks. Looks like I should download and get testing :)
